cmd/ocm/main.go

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/gardener/component-cli/cmd/ocm/app"
+)
+
+func main() {
+	ctx := context.Background()
+	defer ctx.Done()
+	cmd := app.NewComponentsCliCommand(ctx)
```

Comment:
Should we also rename the function from NewComponentsCliCommand() to NewOCMCliCommand() ?